packages/netlify-cms-backend-git-gateway/src/GitHubAPI.ts

```diff
@@ -71,6 +71,20 @@ export default class API extends GithubAPI {
     return Promise.resolve({ login: '', ...this.commitAuthor });
   }
 
+  async getHeadReference(head: string) {
```

Comment:
This is a bit of a hack. When calling [GitHub Pulls API](https://developer.github.com/v3/pulls/#list-pull-requests) you can filter pull requests by head and base. base is just the branch name, but head must include the owner in the format of `user:head` otherwise this parameter will be ignored with no error and all pull requests will be returned.
Another option is to get all pull requests and do some client side filtering.

Reply:
We’ll hopefully be parsing all pull requests anyway, either route is fine for now.